### ISSUE TITLE
Errors in details page for any specific file in HTML report 

### ISSUE THREAD

**Issue by cgalvarez** [serut/meteor-coverage#21]
## Expected Behavior

User should be able to see the coverage details for each file in the HTML report.

## Current Behavior

Right now seems that the package adds the substring `packages/<package-name>` to the `COVERAGE_APP_FOLDER` (which points to the current package folder `/home/threepwood/git/github/meteor-coverage/`), like this: `/home/threepwood/git/github/meteor-coverage/packages/meteor-coverage/server/main.js`, and the file contents can't be fetched to fill the HTML because the filepath does not exist.

This happens for ALL the files in the HTML report.

## Possible Solution

None right now.

## Steps to Reproduce (for bugs)

1. `npm install -g https://github.com/serut/spacejam/tarball/windows-suppport-rc4`
2. `git clone https://github.com/serut/meteor-coverage.git`
3. `cd meteor-coverage`
4. `npm run test`
5. Open the file `.coverage/index.html` after package coverage finishes.
6. Click on any file link, like `main.js`

You'll get a screen similar to this one:

![meteor-package-error](https://cloud.githubusercontent.com/assets/10707639/18319422/e8c14808-7525-11e6-8356-8c00e15e0c4a.png)

## Context

I was facing this same problem when trying to get the coverage for my own package, and thought that I was doing something wrong. Then I tested this package with their own files and configuration, and got the same error.

## Your Environment

- Versions used:
   - meteor@1.4.1.1
   - lmieulet:meteor-coverage@0.9.4
   - practicalmeteor:mocha@2.4.5_6
   - spacejam: https://github.com/serut/spacejam/tarball/windows-suppport-rc4
- Environment name and version: Node.js v6.3.1
- Operating System and version: Ubuntu Xenial Desktop 16.04

**Comment by serut** [comment]
Workaround:
```
meteor create some-sandbox
cd some-sandbox
mkdir packages
cd packages
git clone yourpackage
cd ..
meteor add yourpackage
spacejam test-packages --driver-package practicalmeteor:mocha --coverage out_text_summary 

```
To be able to generate html reports anyway

**Comment by cgalvarez** [comment]
This problem should be fixed with the referenced PR I've just made. Tested with your package and mine.

**Comment by cgalvarez** [comment]
Closing since this issue is being fixed in `dev-1.0.0`
